Automattic/wp-calypso
# Business tools redesign: Update page colors

Resolves https://github.com/Automattic/dotcom-forge/issues/9658

## Proposed Changes

* Updates the background color and link navigation colors as per designs pIiTEGIxHIzKfHv1KHOUM5-fi-3130_24771

|Before | After|
|-------|------|
| ![CleanShot 2024-11-11 at 16 35 12@2x](https://github.com/user-attachments/assets/165c0126-d2e2-4ebf-b3fc-6aed11ee8227)| ![CleanShot 2024-11-11 at 16 35 02@2x](https://github.com/user-attachments/assets/762cf255-c858-4ea2-b325-a1d4d73804df) |


## Why are these changes being made?


* To improve the Business Services offered as described in the parent task https://github.com/Automattic/dotcom-forge/issues/9340

## Testing Instructions



* Apply this patch and navigate to the Business Tools pages in both, the site view and the multi-sites view:
  * /marketing/tools/:siteId
  * /sites/marketing/tools/:siteId
* Make sure the style of the cards matches the designs pIiTEGIxHIzKfHv1KHOUM5-fi-3130_24771

## Pre-merge Checklist
